### ISSUE TITLE
Add OpenStack as an upload target

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/osbuild-composer#876]
#776 added support to upload & boot images in OpenStack. I think we can take advantage of it, adapt the existing code and introduce support for uploading to OpenStack using osbuild-composer.

I think we can benefit from this even in our testing setup - uploading minimal images to be used by our CI sounds very cool. :-)

**Comment by atodorov** [comment]
@ondrejbudai I think we should start with adding the necessary credentials to CI.

**Comment by major** [comment]
Here's how to use the OpenStack creds:

* https://github.com/osbuild/imagebuilder-ci-images/blob/master/schutzbot/Jenkinsfile#L60-L62
* https://github.com/osbuild/imagebuilder-ci-images/blob/master/build-openstack-image.sh#L143-L145

**Comment by ondrejbudai** [comment]
@atodorov do you want to look into the credentials or should I do it?

**Comment by atodorov** [comment]
@ondrejbudai is there something else left to do here ?

**Comment by ondrejbudai** [comment]
Yeah, osbuild-composer still doesn't have OpenStack as an upload target. ;) Therefore, users are currently not able to upload images to OpenStack using osbuild-composer REST APIs.

**Comment by teg** [comment]
Would love to have this, but not top priority at the moment.
